api/src/modules/sections/sections.controller.ts

```diff
@@ -13,8 +13,47 @@ export class SectionsController {
   @TsRestHandler(c.searchSections)
   public async searchSections(): Promise<ControllerResponse> {
     return tsRestHandler(c.searchSections, async ({ query }) => {
-      const data = await this.sectionsService.findAllPaginated(query);
-      return { body: data, status: 200 };
+      const { data, metadata } =
+        await this.sectionsService.findAllPaginated(query);
+      const mockedData = addRandomDataToWidgets(data);
+      return { body: { data: mockedData, metadata }, status: 200 };
     });
   }
 }
+
+/**
+ * @description temporal hack to add random data to widgets until we get access to the real data, or progress with filters
+ */
+
+function addRandomDataToWidgets(sections) {
+  const randomInt = (min, max) =>
+    Math.floor(Math.random() * (max - min + 1)) + min;
+
+  const randomLabel = () => {
+    const labels = [
+      'Value 1',
+      'Value 2',
+      'Value 3',
+      'Value 4',
+      'Value 5',
+      null,
+    ];
+    return labels[randomInt(0, labels.length - 1)];
+  };
+
+  const generateRandomDataObject = () => ({
+    value: randomInt(1, 1000),
+    total: randomInt(1000, 5000),
+    label: randomLabel(),
```

Comment:
It's not a very big deal for now, but since there are no filters yet the value for value and total can be the same.